02-microarray/differential-expression_microarray_01_2-groups.html

```diff
@@ -1998,7 +2000,7 @@ <h1><span class="header-section-number">6</span> Session info</h1>
 sessionInfo()</code></pre>
 <pre><code>## R version 4.0.2 (2020-06-22)
 ## Platform: x86_64-pc-linux-gnu (64-bit)
-## Running under: Ubuntu 20.04.1 LTS
+## Running under: Ubuntu 20.04 LTS
```

Comment:
Yes, it appears we have a couple versions of the docker image floating around. This will hopefully resolve in early oct when 4.0.3 is released and rocker/tidyverse:4.0.2 freezes.

Reply:
So it doesn't listen to us when the Dockerfile says `FROM rocker/tidyverse:4.0.2`?

Reply:
It does, but `rocker/tidyverse:4.0.2` is not yet frozen. So you can have different versions in different caches.

Reply:
Ah so should we try to wait it out or can we/should we do something to freeze it now?

Reply:
I'll file an issue about this.

Reply:
#235